shardy/dialect/sdy/ir/attrs.td

```diff
@@ -809,6 +864,13 @@ def Sdy_AxisRefList : ArrayOfAttr<Sdy_Dialect, "AxisRefList",
                                  "axis_ref_list", "AxisRefAttr"> {
   let assemblyFormat = "`{` (`}`) : ($value^ `` `}`)?";
   let summary = "List of axis refs";
+  let description = [{
+    **Constraints:**
+    - Elements in `value` must satisfy the constraints of `AxisRefAttr`.
+    - There are no duplicate axis-refs.
```

Comment:
Lets expand to:

There are no duplicate axis-refs or sub-axes that overlap with one another.